### ISSUE TITLE
Missing scrollbar in Backend

### ISSUE THREAD

**Issue by webdev2027** [mehrwert/TYPO3-phpMyAdmin#29]
Hello. I use `TYPO3 8.7 LTS` and the latest Ext.`phpmyadmin 5.2.5` on apache `PHP7.1`.
There's no scrollbar in Backend, so it's hard to use phpmyadmin as an extension.

These guys got a bugfix .. it works, so please would you implement this small issue? Thanks.

https://forge.typo3.org/issues/84529  

> Description
> 
> No Scrollbar visible in TYPO3 Backend.
> 
> There is a missing CSS statement in Version 5.2.2 in the File:
> 
> phpmyadmin/Vendor/phpMyAdmin-4.7.8-all-languages/themes/pmahome/css/common.css.php

```
#page_content {
    margin: 0 .5em;
    max-height: calc(100vh - 75px);
    overflow: auto;
}
```

**Comment by abeutel** [comment]
Hey @webdev2027, thanks for pointing me to the Forge issue – I somehow lost track of that. Will check ASAP.

**Comment by abeutel** [comment]
Hi @webdev2027, I just added the suggested fixes and release 5.2.6 (tagged) and uploaded to TER.

**Comment by webdev2027** [comment]
Great, thanks Andreas.
